deploy/contents/install/app/install-utils.sh

```diff
@@ -696,9 +698,17 @@ function parse_options {
     fi
 
     if [ -z $CP_INSTALL_CONFIG_FILE ]; then
-        print_warn "-c|--install-config : path to the installation config not set - default configuration will be used"
-        export CP_INSTALL_CONFIG_FILE="$INSTALL_SCRIPT_PATH/../install-config"
-        mkdir -p $(dirname $CP_INSTALL_CONFIG_FILE)
+        is_data_is_available_in_point_in_time_configuration install-config
+        if [ $? -eq 0 ]; then
+           print_warn "To prevent data changes in $CP_POINT_IN_TIME_CONFIGURATION_DIR/install-config it will be copied to Temp directory before processing"
+           mkdir -p ~/temp/$CP_POINT_IN_TIME_CONFIGURATION_DIR/
```

Comment:
use `mktemp` here to get a temporary file, no need to recreate the whole path structure, we would be file with just a temp file